tests/garage/tf/baselines/test_baselines.py

```diff
@@ -4,36 +4,23 @@
 """
 import tensorflow as tf
 
-from garage.envs.wrappers import Resize
 from garage.tf.baselines import ContinuousMLPBaseline
-from garage.tf.baselines import GaussianConvBaseline
 from garage.tf.baselines import GaussianMLPBaseline
 from garage.tf.envs import TfEnv
 from tests.fixtures import TfGraphTestCase
 from tests.fixtures.envs.dummy import DummyBoxEnv
-from tests.fixtures.envs.dummy import DummyDiscrete2DEnv
 
 
 class TestTfBaselines(TfGraphTestCase):
+
     def test_baseline(self):
         """Test the baseline initialization."""
         box_env = TfEnv(DummyBoxEnv())
         deterministic_mlp_baseline = ContinuousMLPBaseline(env_spec=box_env)
         gaussian_mlp_baseline = GaussianMLPBaseline(env_spec=box_env)
 
-        discrete_env = TfEnv(Resize(DummyDiscrete2DEnv(), width=64, height=64))
-        gaussian_conv_baseline = GaussianConvBaseline(
```

Comment:
Shouldn't this be changed to `GaussianCNNBaseline` instead of removing?

Reply:
This file actually tests nothing. It simply calls the constructor and functions for coverage. The baselines should be tested in their own file, e.g. `test_gaussian_cnn_baseline.py` for `GaussianCNNBaseline`.